crypto/pem/pem_lib.c

```diff
@@ -828,6 +828,12 @@ static int get_header_and_data(BIO *bp, BIO **header, BIO **data, char *name,
             goto err;
         }
 
+        /* Check if line has been read completely or if only part of the line
+         * has been read. Keep the previous value to ignore newlines that
+         * appear due to reading a line up until the char before the newline */
```

Comment:
Style nit - the end of multiline comment should be on a separate line with the `*` aligned.

Reply:
okay, done

Reply:
Shouldn't the opening `/*` also be on a line of its own (for a multiline comment)?